optimize/client/scripts/dependencies.js

```diff
@@ -5,16 +5,15 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
+import fs from 'fs';
```

Comment:
We can destructure these too if we are destructuring everything else
```suggestion
import {readFileSync, writeFileSync} from 'fs';
```